app/models/entities/organization.rb

```diff
@@ -0,0 +1,34 @@
+class Entities::Organization < Maestrano::Connector::Rails::Entity
+  def self.connec_entity_name
+    'Organization'
+  end
+
+  def self.external_entity_name
+    'Organization'
+  end
+
+  def self.mapper_class
+    OrganizationMapper
+  end
+
+  def self.object_name_from_connec_entity_hash(entity)
+    entity['name']
+  end
+
+  def self.object_name_from_external_entity_hash(entity)
+    entity['name']
+  end
+end
+
+class OrganizationMapper
+  extend HashMapper
+
+  map from('name'), to('name')
+  map from('code'), to('company_id')
+  map from('number_of_employees'), to('people_count')
+  map from('address/billing/line1'), to('address_admin_area_level_1')
```

Comment:
'Line1' and 'line2' might be better mapped to  'address_street_number' + 'address_route' and 'address_sublocality'. Or is what 'address_admin_area_level_x' do?